roles/setup_efm/tasks/setup_efm.yml

```diff
@@ -19,6 +42,7 @@
     - include_tasks: rm_efm_install_config.yml
```

Comment:
Maybe it's time to remove this cleanup task and create a new role for the cleanup role.

Reply:
Same as https://github.com/EnterpriseDB/edb-ansible/pull/85#discussion_r546015786